### PR TITLE
Open media sites

### PR DESCRIPTION
**Pull request by 3Wali**
## Ticket no : [TS-19](https://shop-go.atlassian.net/browse/TS-19)

1 . Add media button with icon 

2. When a user clicks on button will open safari and show the  login page

### CODE REVIEW

**Comment by 05Waleed** [comment]
where are images? 

https://github.com/haroonmaqsoodch/ShopGo/assets/139551937/c3a701ee-3dd2-4f7c-9cad-a9516c3315ce

File: ShopGo/Login/LoginViewController.swift [hunk 1]
```diff
@@ -21,4 +21,18 @@ class LoginViewController: UIViewController {
     
     @IBAction func loginButtonAction(_ sender: UIButton) {
     }
+    
+    @IBAction func facebookButton(_ sender: Any) {
+        UIApplication.shared.open(URL(string:"https://www.facebook.com")! as URL , options: [:] , completionHandler: nil)
```
Comment (haroonmaqsoodch) [inline ShopGo/Login/LoginViewController.swift]: create a function for this instead of writing the code again and again

Comment (3Wali) [reply ShopGo/Login/LoginViewController.swift]: fixed

**Review by zuhair5 — CHANGES_REQUESTED** [review]
- Social media site images are not being shown so fix that
- Stack view with the button images in it has issues due to auto layout not being done correctly.
- Kindly add the button images programmatically
- Merge conflicts are being shown below, kindly resolve them
- Fix the above-mentioned issues and commit again

 **screenshots**
Screenshot 2023-08-02 at 3 20 44 AM
Screenshot 2023-08-02 at 3 21 58 AM

**Comment by 3Wali** [comment]
Fixed

**Review by haroonmaqsoodch — COMMENTED** [review]
Resolve the conflicts before you push your next commit

**Review by zuhair5 — APPROVED** [review]
Seems good to me just wait for haroon bhai to check it out again before merging.

**Pull request merged**